src/knowledge_bases/knowledge_base.ts

```diff
@@ -0,0 +1,181 @@
+import SqlQueryResult from '../sql/sqlQueryResult';
```

Comment:
@md-abid-hussain please double check in types 'any' and using a safer alternative as 'unknown'